toolkit/resources/manifests/package/pkggen_core_aarch64.txt

```diff
@@ -217,7 +217,6 @@ npth-1.6-4.cm2.aarch64.rpm
 pinentry-1.2.0-1.cm2.aarch64.rpm
 gnupg2-2.3.3-1.cm2.aarch64.rpm
 gpgme-1.16.0-1.cm2.aarch64.rpm
-mariner-repos-2.0-2.cm2.noarch.rpm
```

Comment:
I believe you'll need to update the [`toolkit/resources/manifests/package/update_manifests.sh`](https://github.com/microsoft/CBL-Mariner/blob/main/toolkit/resources/manifests/package/update_manifests.sh) file to be consistent with this change.

Reply:
With the suggestion being to add a line after line 82 so that file reads:

```bash
sed -ri '/mariner-repos-(extras|ui|microsoft)/d' $TmpPkgGen
sed -ri '/mariner-repos-[0-9]/d' $TmpPkgGen
```